src/csharp/CSharpParser.g4

```diff
@@ -384,7 +384,8 @@ query_continuation
 
 //B.2.5 Statements
 statement
-    : labeled_statement                                              #labeledStatement
+    : ';'                                                            #emptyStatement
```

Comment:
My bad. I failed somewhere when rebased. I'll just rewrite and force push the original commit with this fix.